indigoTech.js

```diff
@@ -1,5 +1,7 @@
 function indigoTech(farmers, data, code){
```

Comment:
- Missing `'use strict';` at top of file
- Unused variables - `farmers`, `data` and `code`